source/aggregation-pipeline-builder.txt

```diff
@@ -14,21 +14,28 @@ Aggregation Pipeline Builder
 
 *New in version 1.14.0*
 
-The Aggregation Pipeline Builder in |compass| provides the ability to
-create :manual:`aggregation pipelines </core/aggregation-pipeline/>` to
-process data. In aggregation pipelines, documents in a collection
-or view pass through stages where |compass| processes them into a
-set of aggregated results. You can change the stages and results to
-suit your needs.
-
-To start building an aggregation pipeline for a :doc:`collection or view
-</collections>`, choose the collection and click the
-:guilabel:`Aggregations` tab.
-A blank aggregation pipeline displays. The :guilabel:`Preview of
-Documents in the Collection` section of the :guilabel:`Aggregations`
-view displays 20 documents sampled from the current collection.
-
-You can:
+The Aggregation Pipeline Builder in |compass| lets you create
+:manual:`aggregation pipelines </core/aggregation-pipeline/>` to process
+data. In aggregation pipelines, documents in a collection or view pass
+through stages which compute or modify document fields. For example, you
```

Comment:
Can we really change document fields? I have in mind out and merge stages, is it something that you mean?

Reply:
Thx for updating it! 👍